scss/variables/typography.js

```diff
@@ -8,9 +8,10 @@
 	}
 }( this, function( chassis ) {
 
-var color = "#222",
+var color = "map-get($text, base)",
 	fontSize = "20px",
-	lineHeight = 1.5;
+	lineHeight = 1.5,
+	linkColor = "map-get($primary, light)";
```

Comment:
might want to have links be a variation on the primary color scheme based on active/hover/focus state? similar to how buttons work